content/2017/1/11/typography.rst

```diff
@@ -25,6 +25,10 @@ after:
 How to Choose Engaging & Accessible Typography for Your Website Brand
 =====================================================================
 
+.. callmacro:: blog/macros.j2#byline
+   get_blog_entries_by_bird('kit')[0]
```

Comment:
yeah, that's certainly possible, but like I said, at first glance it just appears to use it for `__getitem__`, not `__iter__/keys()`